src/algebra/lie/cartan_matrix.lean

```diff
@@ -147,7 +168,98 @@ end cartan_matrix
 
 Note that it is defined for any matrix of integers. Its value for non-Cartan matrices should be
 regarded as junk. -/
-@[derive [lie_ring, lie_algebra R]]
+@[derive [inhabited, lie_ring, lie_algebra R]]
 def matrix.to_lie_algebra := (cartan_matrix.relations.to_ideal R A).quotient
 
-instance (A : matrix B B ℤ) : inhabited (matrix.to_lie_algebra R A) := ⟨0⟩
+namespace cartan_matrix
+
+/-- The Cartan matrix of type e₆. See `bourbaki1968` plate V, page 277.
```

Comment:
Usually the Dynkin type is capitalized, right? Is there a reason that you use lowercase beyond Lean conventions? I would be happy to break with the convention here (-;

Reply:
No reason, also happy to break the convention ;-)

Done!